config/http/handler/simple.json

```
@@ -11,6 +11,7 @@
       "handlers": [
         { "@id": "urn:solid-server:default:Middleware" },
         {
+          "@id": "urn:solid-server:simple:BaseHttpHandler",
```

```suggestion
          "@id": "urn:solid-server:default:BaseHttpHandler",
```

In general the idea is to keep the identifiers the same for different import options of the same category for easier reuse.